docs/applications/containers/how-to-deploy-the-elastic-stack-on-kubernetes/index.md

```diff
@@ -287,36 +291,73 @@ In addition to collecting logs with Filebeat, Metricbeat can collect pod and nod
 
 1.  At least one `metricbeat` index should be present, similar to the following:
 
-        green open metricbeat-7.3.0-2019.08.02-000001 N75uVk_hTpmVbDKZE0oeIw 1 1   455  0   1.1mb 567.9kb
+        green open metricbeat-7.3.2-2019.09.30-000001 N75uVk_hTpmVbDKZE0oeIw 1 1   455  0   1.1mb 567.9kb
 
 ### Load Dashboards
 
 Metricbeat can install default Dashboards into Kibana to provide out-of-the-box visualizations for data collected by Kubernetes.
 
-Before following these steps, ensure that the `port-forward` command to expose Kibana over port `5601` locally is running.
+Before following these steps, ensure that the `port-forward` command to expose Kibana over port `5601` locally is still running.
+
+Run the following commands on your local machine. This will communicate with Kibana over `127.0.0.1:5601` to import default Dashboards that will be populated by data from Metricbeat.
+
+{{< note >}}
+Your commands should use the same version of Metricbeat deployed to your Kubernetes cluster. You can find this version by issuing the following command:
+
+    helm get values --all metricbeat | grep imageTag
+{{< /note >}}
+
+#### For Linux
+
+1.  Get the Metricbeat package.
+
+        wget https://artifacts.elastic.co/downloads/beats/metricbeat/metricbeat-7.3.2-linux-x86_64.tar.gz
+
+1.  Unzip the package.
+
+        tar xvzf metricbeat-7.3.2-linux-x86_64.tar.gz
+
+1.  Navigate to the directory.
+
+        cd metricbeat-7.3.2-linux-x86_64
+
+1.  Setup the dashboards.
+
+        ./metricbeat setup --dashboards
+
+#### For MacOS
+
+1.  Get the Metricbeat package.
+
+        wget https://artifacts.elastic.co/downloads/beats/metricbeat/metricbeat-7.3.2-darwin-x86_64.tar.gz
+
+1.  Unzip the package.
+
+        tar xvzf metricbeat-7.3.2-darwin-x86_64.tar.gz
+
+1.  Navigate to the directory.
```

Comment:
Thanks for this - it worked great! I removed the docker command line and split this section into linux and Mac command sections.